wiki/Getting-Started.md

```diff
@@ -164,14 +164,14 @@ If you're not on NixOS, you may need [NixGL](https://github.com/nix-community/ni
 nix run --impure github:guibou/nixGL -- ./results/bin/niri
 ```
 
-### Installation
+### Packaging
 
-The recommended way to install and run niri is as a standalone desktop session.
-To do that, put files into the correct directories according to this table.
+The recommended way to package niri  is to have niri run as a standalone desktop
+session. To do that, put files into the correct directories according to this table.
```

Comment:
```suggestion
The recommended way to package niri is so that it runs as a standalone desktop session.
To do that, put files into the correct directories according to this table.
```